data/software.json

```diff
@@ -387,6 +387,17 @@
         "platforms": [],
         "url": null
     },
+    {
+        "categories": [
+            "client"
+        ],
+        "doap": "https://codeberg.org/prav/prav/raw/branch/master/prav.doap",
```

Comment:
This file does not exist.

If you provide a DOAP file, `platforms` and `url` won't need to be filled.

Reply:
File has been created. Made a new merge request https://github.com/xsf/xmpp.org/pull/1450